src/main/java/ProjectOcean/Controller/RemoveMovableChild.java

```diff
@@ -0,0 +1,6 @@
+package ProjectOcean.Controller;
+
+@FunctionalInterface
+public interface RemoveMovableChild {
+    void removeMovableChild(Movable movable);
```

Comment:
javadoc

Reply:
Not my interface, so let someone else do this, will write it onto refactor doc